modules/veteran/app/models/veteran/base.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+require 'evss/auth_headers'
+
+# Veteran model
+module Veteran
+  class Base < Common::Base
```

Comment:
Is your plan to inherit this other places? Otherwise I'm not sure base is a good name

Reply:
Also given the `ClaimsApi::Veteran` This could get really confusing really fast. Seems like that should be incorporated, or at least made more clear

Reply:
So I gave some thought to this and can change it to Veteran::User if that makes more syntactical sense here. I tend to default to Base classes when I'm planning on using them as inheritance objects in the future but can make the change if you feel User is a better term.

My goal will be to eventually migrate the ClaimsApi::Veteran to Veteran::Base or Veteran::User but since this PR is just to handle the module creation and migration I can include this to the required tasks for my next PR implementing the business logic behind these.

Reply:
I think that makes the most sense. 

As a side note I feel like this a breakdown of the system of doing migration tasks separate. It feels like merging in unfinished code for the sake of the migration. It's either a flawed system or we're doing it wrong.

Reply:
The separation is less about merging unfinished code and more about not merging anything that would break something until the migration is complete so in this case I wouldn't reference the Organization or Representative models in a controller for example until the model and the migration is complete ensuring that if the deploy happens before the migration is ran you don't end up getting a 500 response from a functioning end point like say Claims for example.

I went ahead and renamed it Veteran::User with Veteran::Base still in place so we can use it for ClaimsApi::Veteran in the refactor.